src/jupytext/cell_reader.py

```diff
@@ -555,6 +555,11 @@ def uncomment_code_and_magics(self, lines):
                 else:
                     lines = uncomment(lines)
 
+        if self.default_language == "go" and self.language is None:
+            lines = [
+                re.sub(r"^([/\s]*)(//\s*gonb:%%)", r"\1%%", line) for line in lines
```

Comment:
We need to escape the `%%` commands in Markdown cells too, that's why we might find multiple levels of comments.